dbus-codegen/src/generate.rs

```diff
@@ -80,13 +81,16 @@ pub struct GenOpts {
     pub futures: bool,
     /// Type of connection, for client only
     pub connectiontype: ConnectionType,
+    /// trait filter. Only matching traits are generated, if non-empty.
```

Comment:
Call this "interface filter" instead of "trait filter", as it matches on the D-Bus interface name, not the generated trait name (as it should be).

It should be an `Option<HashSet<String>>` so that `None` matches everything and `Some(HashSet::new())` matches nothing.

Also, a better name for the variable would be `interfaces` or  `interface_filter`.